pulp_ansible/app/galaxy/v3/views.py

```diff
@@ -99,10 +101,8 @@ def retrieve(self, request, *args, **kwargs):
         Returns a CollectionVersion object.
         """
         instance = self.get_object()
-        artifact = ContentArtifact.objects.get(content=instance)
 
         context = self.get_serializer_context()
-        context["content_artifact"] = artifact
```

Comment:
@bmbouter `CollectionVersionViewSet` only presented `artifact` data on `retrieve`, so the strategy was getting the `ContentArtifact` and injecting it into the context.
For `UnpaginatedCollectionVersionViewSet` we need to present `artifact` data on `list`, so I changed the query to select related `ContentArtifact`

Reply:
This is pretty slick. :+1: